modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/di/ServersModule.kt

```diff
@@ -247,7 +243,7 @@ class ServersModule {
     @Provides
     @SyncServerRetrofit
     @Singleton
-    internal fun provideApiRetrofit(@CachedOkHttpClient okHttpClient: OkHttpClient, moshi: Moshi): Retrofit {
+    internal fun provideApiRetrofit(@NoCacheOkHttpClient okHttpClient: OkHttpClient, moshi: Moshi): Retrofit {
```

Comment:
Do you remember why we needed this change?

Reply:
I should have left a comment on this change. I couldn't understand why we could use a cache with the User Sync API calls. As there are lots of other changes happening in this pull request I'm going to revert this.

Reply:
>  I couldn't understand why we could use a cache with the User Sync API calls

The cached client probably still respects the cache-control headers. I was fine with the change (it made sense because it seemed like we wouldn't want caching here), I was more just curious if there was a reason we now had to make the change now—something like one of the new calls not including the right cache-control headers in the response or something.